pipeline_games/test_transform.py

```diff
@@ -0,0 +1,69 @@
+"""Testing file for transform script"""
+import pytest
+from pandas._libs.tslibs.timestamps import Timestamp
+
+from transform_games import identify_unique_tags, create_user_generated_column, drop_unnecessary_columns, convert_date_to_datetime, convert_price_to_float, explode_column_to_individual_rows
+
+
+def test_separate_rows_created_for_unique_tags(fake_raw_data):
+    """Tests that unique tags have a separate row"""
+    assert 'tags' not in list(fake_raw_data.columns)
+    assert fake_raw_data.shape[0] == 1
+    result = identify_unique_tags(fake_raw_data)
+    assert result.shape[0] == 4
+    assert 'tags' in list(result.columns)
+
+
+def test_user_generated_column_created(fake_data_with_tags):
+    """Check new column added to dataframe"""
+    assert len(fake_data_with_tags.columns) == 13
+    result = create_user_generated_column(fake_data_with_tags)
+
+    assert len(result.columns) == 14
+
+
+def test_columns_dropped(fake_data_with_tags):
+    """Check specified column is dropped"""
+    assert len(fake_data_with_tags.columns) == 13
+    assert 'user_tags' in list(fake_data_with_tags.columns)
+    result = drop_unnecessary_columns(fake_data_with_tags, 'user_tags')
+    assert len(result.columns) == 12
+    assert 'user_tags' not in list(result.columns)
+
+
+def test_date_converted_if_valid(fake_date):
```

Comment:
Maybe this could be two separate tests

Reply:
When you say two separate tests is this one for seeing it's a timestamp and another to see it's the correct result?

Reply:
Disagree with Dan, since these are testing on the exactly the same dataframe